styles/alert.css

```
@@ -22,3 +22,102 @@
   --bs-alert-color: rgb(var(--stanford-illuminating-dark-rgb));
   --bs-alert-border-color: rgb(var(--stanford-illuminating-dark-rgb));
 }
+
+/* Banner styles */
+.banner {
+  --banner-bg: none;
+  --banner-padding-x: 16px;
+  --banner-padding-y: 16px;
+  --banner-margin-bottom: 1rem;
```

Why are you mixing units?
```suggestion
  --banner-padding-x: 1rem;
  --banner-padding-y: 1rem;
  --banner-margin-bottom: 1rem;
```